provisionerd/proto/version.go

```diff
@@ -4,7 +4,7 @@ import "github.com/coder/coder/v2/apiversion"
 
 const (
 	CurrentMajor = 1
-	CurrentMinor = 1
+	CurrentMinor = 2
```

Comment:
Can you add a "version history" comment like in: https://github.com/coder/coder/blob/3f1795fd92d1352afa3f2d467923d8101a1f283d/tailnet/proto/version.go#L7